Descriptors/AcademicSubjectDescriptor.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<InterchangeDescriptors xmlns="http://ed-fi.org/3.3.1-b" xmlns:ann="http://ed-fi.org/annotation" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://ed-fi.org/3.3.1-b ../Schemas/Bulk/EXTENSION-Interchange-Descriptors-Extension.xsd">
+	<AcademicSubjectDescriptor>
+		<CodeValue>Bilingual</CodeValue>
+		<ShortDescription>Bilingual</ShortDescription>
+		<Description>Bilingual</Description>
+		<Namespace>uri://tpdm.ed-fi.org/AcademicSubjectDescriptor</Namespace>
+	</AcademicSubjectDescriptor>
+	<AcademicSubjectDescriptor>
+		<CodeValue>ELL</CodeValue>
```

Comment:
If possible, "ELL" should be spelled out fully - the Ed-Fi practice is to avoid acronyms and abbreviations in such values.